src/ui/wxWidgets/MenuFileHandlers.cpp

```diff
@@ -875,7 +875,7 @@ void PasswordSafeFrame::DoImportText(wxString filename)
     // Database is not unique to start with - tell user to validate it first
     wxMessageBox(wxString() << _("The database:") << wxT("\n\n") << m_core.GetCurFile() << wxT("\n\n")
                             << _("has duplicate entries with the same group/title/user combination.")
-                            << _("  Please fix by validating database."),
+                            << _("  Fix by validating database."),
```

Comment:
Side note: we shouldn’t be conctenating translatable strings like this because other languages have different word orders. We should be using format placeholders instead.

Reply:
Agree, but that's another PR...